CGRPCZlib.podspec

```diff
@@ -0,0 +1,19 @@
+Pod::Spec.new do |s|
+
+    s.name = 'CGRPCZlib'
+    s.module_name = 'CGRPCZlib'
+    s.version = '1.0.0-alpha.11'
+    s.license = { :type => 'Apache 2.0', :file => 'LICENSE' }
+    s.summary = 'Swift gRPC code generator plugin and runtime library'
```

Comment:
Should CGRPCZlib have a different summary?

Reply:
It can, I was thinking having something more generic that might fit both would keep the generation script simple. What do you think @MrMage? I'm open to either way 👍

Reply:
Personally I would use a slightly different description for each, to avoid confusing anyone who looks up either package at the CocoaPods site. I understand that would need a bit of additional code, though.

Reply:
I agree, that does make sense. Would you mind if I took that one up as a follow-up item and address it separately?